source/gui/settingsDialogs.py

```diff
@@ -1991,6 +1991,16 @@ def makeSettings(self, settingsSizer):
 		self.emphasisCheckBox=fontGroup.addItem(wx.CheckBox(self,label=emphasisText))
 		self.emphasisCheckBox.SetValue(config.conf["documentFormatting"]["reportEmphasis"])
 
+		# Translators: This is the label for a checkbox in the
+		# document formatting settings panel.
+		highlightText = _("Highlight")

```

Comment:
oh, just noticed this. Ignore my comment suggesting we could add a specific option.

Though, I think it should use the same term "Marked" rather than highlight. If the concern is that people may not know the intention of marked (and we want to stick with announcing "marked") then this could be "Marked - for highlighted text" an explanation in the user guide would also help to explain the purpose of this feature.